examples/plugins/transform/pipeline_operator_plugin.cpp

```diff
@@ -6,20 +6,20 @@
 // SPDX-FileCopyrightText: (c) 2021 The VAST Contributors
 // SPDX-License-Identifier: BSD-3-Clause
 
+#include <vast/pipeline_operator.hpp>
 #include <vast/plugin.hpp>
-#include <vast/transform_step.hpp>
 
```

Comment:
Just commenting somewhere random again to leave a resolvable note: We need to rebase this after merging #2417 and then rename again as necessary.